requirements-full.txt

```diff
@@ -162,15 +163,15 @@ python-dotenv==0.19.0; python_version >= "3.5"
 pytrends==4.7.3
 pytz==2021.1; python_full_version >= "3.6.8" and python_version >= "3.6" and python_full_version < "4.0.0"
 pyupgrade==2.18.2; python_full_version >= "3.6.1"
-pywin32==300; sys_platform == "win32" and python_version >= "3.6"
+pywin32==301; sys_platform == "win32" and python_version >= "3.6"
```

Comment:
you need to regenerate requirements-full.txt

Reply:
I regenerated after running poetry update them, but now I can't get poetry to revert pywin32 `Unable to find installation candidates for pywin32 (300)`.  

pip install pywin32==300 also fails, so I may just close this for now.

Reply:
I can do a big dependency update PR.